tests/integration/models/reddit/test_collections.py

```diff
@@ -138,6 +138,31 @@ def test_update_description(self, _):
             collection.mod.update_description(new_description)
             assert new_description == collection.description
 
+    @mock.patch("time.sleep", return_value=None)
+    def test_update_display_layout(self, _):
+        self.reddit.read_only = False
+        uuid = self.NONEMPTY_REAL_UUID
+        gallery_layout = "GALLERY"
+        lowercase_gallery_layout = "gallery"
+        timeline_layout = "TIMELINE"
+        empty_string = ""
+        something_random = "colossal atom cake"
+        with self.use_cassette():
+            collection = self.subreddit.collections(uuid)
+            collection.mod.update_display_layout(gallery_layout)
+            assert gallery_layout == collection.display_layout
+            collection.mod.update_display_layout(timeline_layout)
+            assert timeline_layout == collection.display_layout
+            collection.mod.update_display_layout(lowercase_gallery_layout)
+            assert lowercase_gallery_layout != collection.display_layout
+            assert timeline_layout == collection.display_layout
+            collection.mod.update_display_layout(empty_string)
+            assert empty_string != collection.display_layout
+            assert timeline_layout == collection.display_layout
+            collection.mod.update_display_layout(something_random)
+            assert something_random != collection.display_layout
+            assert timeline_layout == collection.display_layout
```

Comment:
These will need to be separate tests. I would recommend having the following tests:
- `test_update_display_layout`: this tests both types and asserts that it is properly updated.
- `test_update_display_layout__empty_string`: `collection.mod.update_display_layout("")` 
- `test_update_display_layout__invalid_layout`: see what happens if an invalid layout is set and assert that the expected exception is thrown.
- `test_update_display_layout__lowercase`: same as above and also assert that the expected exception is thrown.
- `test_update_display_layout__none`: `collection.mod.update_display_layout(None)`

Reply:
Thanks for the suggested changes. I'll get onto them, but do you know how I could resolve the 401 error? Any tips? It'd really help with the unit tests.

Reply:
The 401 errors are due to missing credentials. See my [comment here](https://github.com/praw-dev/praw/pull/1829#discussion_r774758407) for how to set environment variables for your current session.